packages/extension/src/newtab/ShortcutLinks.tsx

```diff
@@ -29,6 +42,21 @@ export default function ShortcutLinks({
     onSaveChanges,
   } = useShortcutLinks();
 
+  const mountedRef = useRef(false);
+
+  useEffect(() => {
```

Comment:
Shouldn't we check for showTopSites? Otherwise, we may track this impression without actually showing it

Reply:
It says in taxonomy sheet it should fire when component renders?

Reply:
Do you want it only when some shortcuts are actually rendered?

Reply:
I raised this question on taxonomy as well :) 
It basically goes for everything, do we want to track it in general or only for custom, browser ones, etc?

Reply:
Sorry my question was confusing. I meant we need to make sure we actually render the shortcuts. Because I see right after the `useEffect` a condition to make sure we should render it

Reply:
Got it, fixed.